scripts/ci_tests/rust_tests.sh

```diff
@@ -89,6 +89,29 @@ function mk_test_filterset() {
     echo "${FILTERSET}"
 }
 
+function mk_exclude_filterset() {
+    EXCLUDE_SET=""
+
+    # These require extra config
```

Comment:
can we just add what config and why they require more config here ? are they the tests using postgres ? do we have any links to relevant readmes we can add here ?

Reply:
Sure, I'll elaborate

Reply:
saw the changes, thank you !